crates/bevy_render/src/shader/shader.rs

```diff
@@ -64,14 +81,16 @@ impl Into<shaderc::ShaderKind> for ShaderStage {
     }
 }
 
-#[cfg(target_os = "ios")]
+#[cfg(any(target_os = "ios", all(target_arch = "aarch64", target_os = "macos")))]
 pub fn glsl_to_spirv(
     glsl_source: &str,
     stage: ShaderStage,
     shader_defs: Option<&[String]>,
 ) -> Result<Vec<u32>, ShaderError> {
-    let mut compiler = shaderc::Compiler::new()?;
-    let mut options = shaderc::CompileOptions::new()?;
```

Comment:
These lines were not compiling prior to my pull request. Both of these lines return `Option`, whereas the function signature returns `Result`.

Reply:
Thanks for fixing it :heart: